src/bundle/Resources/public/scss/_inputs.scss

```
@@ -69,6 +98,33 @@
     }
 
     &--radio {
-        @include radio-input;
+        width: calculateRem(16px);
+        height: calculateRem(16px);
+        border-radius: calculateRem(8px);
+
+        &:checked {
+            box-shadow: inset 0 0 0 calculateRem(4px) $ibexa-color-primary;
+
+            &:disabled {
+                border-color: $ibexa-color-primary-300;
+                box-shadow: inset 0 0 0 calculateRem(4px) $ibexa-color-primary-300;
+            }
+
+            &:active,
+            &:focus {
+                &:after {
```

```suggestion
                &::after {
```
ref: https://developer.mozilla.org/en-US/docs/Web/CSS/::after#syntax